File: alcance-backend/gestion_presupuesto/comando/comando-aplicacion/src/main/java/com/ceiba/core/comando/ComandoSprint.java
package com.ceiba.core.comando;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

import java.time.LocalDateTime;
import java.util.List;

@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
public class ComandoSprint {
    private Long id;
    private String nombre;
    private LocalDateTime fechaInicial;
    private LocalDateTime fechaFinal;
    private Integer numeroPersonas;
    private Long idProyecto;
    private List<ComandoPresupuestoSprint> presupuestoSprint;
}

File: alcance-frontend/src/app/feature/sprint/shared/model/comando-sprint.ts
import { PresupuestoSprint } from './presupuesto-sprint';

export class ComandoSprint {
  id: number;
  nombre: string;
  fechaInicial: string;
  fechaFinal: string;
  numeroPersonas: number;
  idProyecto: number;
  presupuestoSprint: PresupuestoSprint[];

  constructor(id: number, nombre: string, fechaInicial: string, fechaFinal: string, numeroPersonas: number,
              idProyecto: number, presupuestoSprint: PresupuestoSprint[]) {
      this.id = id;
      this.nombre = nombre;
      this.fechaInicial = fechaInicial;
      this.fechaFinal = fechaFinal;
      this.numeroPersonas = numeroPersonas;
      this.idProyecto = idProyecto;
      this.presupuestoSprint = presupuestoSprint;
  }
}
